### DISCUSSION TITLE
Document Compiling a New C/C++ Module to WebAssembly with WASI

### DISCUSSION THREAD

**Discussion by kzhsw** [mdn/mdn-community#508] [Content]
In page [Compiling a New C/C++ Module to WebAssembly](https://developer.mozilla.org/en-US/docs/WebAssembly/C_to_wasm), Emscripten SDK was chosen. The [WASI sdk](https://github.com/WebAssembly/wasi-sdk) is also possible to compile many C/C++ Module with a standardized system interface and less js glue code, maybe this can also be documented here.

**Comment by bsmth** [comment]
Thanks for raising. I think it's better to track in a content issue here: https://github.com/mdn/content/issues/37587 so I'm closing this for now.
